### ISSUE TITLE
SolO IRFU quicklooks: Unrecognized function or variable 'timet_ici'.

### ISSUE THREAD

**Issue by ErikPGJ** [irfu/irfu-matlab#102]
### Step 1: Latest code?
- [X] This issue is present in the most recent code (i.e. after running ```git pull``` this is still a problem).
  Used 2df7d9a9.

### Step 2: Describe your environment
 * irfu-matlab branch (e.g. ```master```, ```devel```): devel
 * Matlab version used (e.g. ```Matlab R2020b```): MATLAB R2022b
 * Operating system (e.g. ```Windows 10```, ```Mac OS X 10.15.6```, ```Linux Ubuntu 20.04```): Ubuntu 18.04.6 LTS
 - [X] A 64-bit installation of Matlab was used.

### Step 3: Describe the problem

Crashes when making the call `solo.quicklook_main('/home/erjo/so_irfu-matlab_qli//mission/solar_orbiter/+solo/irf_logo.png', '/data/solo/data_yuri/', '/home/erjo/temp/so_qli/so_qli_MTEST.2022-08-30_15.59.18.0_1_2020-04-15T00:00:00---2020-04-25T00:00:00', '0', '1', '2020-04-15T00:00:00', '2020-04-25T00:00:00');
`.
  ```
Operating system is OK
Matlab version is OK
CDF_LEAPSECONDSTABLE is OK
saving caching status
DB caching enabled
  [warning: solo_db.get_variable/append_sci_var(131)] Discarded 1 data points
  [warning: solo_db.get_variable/append_sci_var(131)] Discarded 1 data points
  [warning: solo_db.get_variable/append_sci_var(131)] Discarded 1 data points
  [warning: solo_db.get_variable/append_sci_var(131)] Discarded 1 data points
  [warning: solo_db.get_variable/append_sci_var(131)] Discarded 1 data points
  [warning: solo_db.get_variable/append_sci_var(131)] Discarded 1 data points
  [warning: solo_db.get_variable/append_sci_var(131)] Discarded 1 data points
  [warning: solo_db.get_variable/append_sci_var(131)] Discarded 1 data points
[critical: read_TNR(116)] no data at all ?!?
exception.message=Unrecognized function or variable 'timet_ici'.
read_TNR, row 119
quicklooks_7days, row 130
quicklook_main, row 317
  ```

**Comment by ErikPGJ** [comment]
Solved with 844567b4 (or earlier)?

**Comment by ErikPGJ** [comment]
Test run using 555610 (commit is not the bugfix) implies that bug has been fixed.
